ui-particles-angular/projects/ui-particles-angular/src/lib/oem-theme/gio-top-bar/gio-top-bar.stories.ts

```diff
@@ -91,6 +91,22 @@ export const Default: Story = {
               <div class="image"></div>
             </gio-top-bar-menu>
           </gio-top-bar>
+          <gio-top-bar>
+            <button mat-icon-button>
+            <img src="${args.logo}" alt="custom logo" style="max-height: 36px; max-width: 36px">
```

Comment:
What about adding the logo to the args used for all the stories? It could show what the logo changes for each component.

Reply:
Done. I have renamed the constant to make it more explicit